src/interactor.tsx

```diff
@@ -230,42 +230,42 @@ function Main (props) {
               : null
             }
           </Form.Field>
-        )}
-        <Form.Field style={{ textAlign: 'center' }}>
+        ) }
+        <Form.Field style={ { textAlign: 'center' } }>
           <InteractorSubmit
-            accountPair={accountPair}
-            setStatus={setStatus}
-            attrs={{ interxType, palletRpc, callable, inputParams, paramFields }}
+            accountPair={ accountPair }
+            setStatus={ setStatus }
+            attrs={ { interxType, palletRpc, callable, inputParams, paramFields } }
           />
         </Form.Field>
-        <div style={{ overflowWrap: 'break-word' }}>{status}</div>
+        <div style={ { overflowWrap: 'break-word' } }>{ status }</div>
       </Form>
     </Grid.Column>
   );
 }
 
-function InteractorSubmit (props) {
+function InteractorSubmit (props: any) {
```

Comment:
You can use the VSCode to infer all types instead any, it could help. After that to can refactor to some type.

To do that, just remove any and let VSCode complain and will suggest it. "Infer all types"